### ISSUE TITLE
Uncaught TypeError: Cannot read property 'prototype' of undefined

### ISSUE THREAD

**Issue by borisco** [seveibar/text-pastry#15]
[Enter steps to reproduce below:]

1. Just installed the package

**Atom Version**: 1.0.0
**System**: Mac OS X 10.10.3
**Thrown From**: [text-pastry](https://github.com/seveibar/text-pastry) package, v0.4.3

### Stack Trace

Uncaught TypeError: Cannot read property 'prototype' of undefined

```
At /Users/boris/.atom/packages/text-pastry/lib/text-pastry-view.coffee:4

TypeError: Cannot read property 'prototype' of undefined
  at __extends (/Users/boris/.atom/packages/text-pastry/lib/text-pastry-view.coffee:4:199)
  at /Users/boris/.atom/packages/text-pastry/lib/text-pastry-view.coffee:9:5
  at Object.<anonymous> (/Users/boris/.atom/packages/text-pastry/lib/text-pastry-view.coffee:94:5)
  at Object.<anonymous> (/Users/boris/.atom/packages/text-pastry/lib/text-pastry-view.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/boris/.atom/packages/text-pastry/lib/text-pastry.coffee:1:18)
  at Object.<anonymous> (/Users/boris/.atom/packages/text-pastry/lib/text-pastry.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:709:34)
  at Package.module.exports.Package.activateConfig (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:257:12)
  at PackageDetailView.module.exports.PackageDetailView.activate (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:163:26)
  at PackageDetailView.module.exports.PackageDetailView.initialize (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:146:12)
  at PackageDetailView.View (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at new PackageDetailView (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:38:54)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/settings-view.js:256:22
  at SettingsView.module.exports.SettingsView.getOrCreatePanel (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/settings-view.js:261:19)
  at SettingsView.module.exports.SettingsView.showPanel (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/settings-view.js:306:24)
  at space-pen-div.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-card.js:260:26)
  at space-pen-div.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at space-pen-div.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
     -0:20.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
Sublime-Style-Column-Selection, v1.3.0
atom-beautify, v0.28.7
minimap, v4.10.2
tab-numbers, v0.2.0

# Dev
No dev packages
```

**Comment by dsifford** [comment]
See #12 to fix this -- I was having the same issue and this resolved it.

**Comment by borisco** [comment]
Thanks
But it's not working for me :/
I'll have a look later

**Comment by dsifford** [comment]
@borisco -- My apologies; After writing my reply, I noticed that the feature that I was looking for (ctrl-u to undo selection) was something that was added in the 1.0 build of Atom (doh!). So yes, that fix did not work. (sorry for the confusion!)

**Comment by seveibar** [comment]
fixed with #17 thanks for reporting @borisco!
